grid/chat/messages/profile.supr

```diff
@@ -0,0 +1,278 @@
+name = "chat/messages"
+version = "1.0.0"
+
+"""
+Get chat messages
+Retrieve paginated list of messages
+"""
+usecase GetMessages {
+	input {
+		"""
+		Destination
+		Representation of a messaging destination. Can be ID or channel name, depending on the provider capability.
+		"""
+		destination! string!
+		
+		"""
+		After Date
+		Retrieve only messages after this date and time, UNIX timestamp format (e.g. 1095379198.75) 
+		"""
+		afterDate number
+		
+		"""
+		Before Date
+		Retrieve only messages before this date and time, UNIX timestamp format (e.g. 1095379198.75)
+		"""
+		beforeDate number
+
+		"""
+		Limit
+		Maximum number of messages to retrieve in one page
+		"""
+		limit number
+		
+		"""
+		Page
+		Identification of page with paginated results (cursor)
+		"""
+		page string!
+	}
+	
+	result Messages
+	error Error
+
+	example Successful {
+		input {
+			destination = 'general',
+			limit = 2
+		}
+
+		result {
+			messages = [
+				{
+					id = '1'
+					author = {
+						id = 'U01'
+						username = 'user1'
+					},
+					createdAt = 1643835553.658259,
+					text = 'test 1'
+				}
+				{
+					id = '2'
+					author = {
+						id = 'U02'
+						username = 'user2'
+					},
+					createdAt = 1643835555.648069
+					text = 'test 2'
+				}
+			]
+		}
+	}
+
+	example Failed {
+		input {
+			destination = 'not-existing',
+		}
+
+		error {
+			title = 'channel_not_found'
+		}
+	}
+}
+
+model Attachment {
+	"""
+	Id
+	Unique identifier of the attachment
+	"""
+	id! string!
+
+	"""
+	Created At
+	UNIX timestamp (e.g. 1095379198.75) when the attachment was created
+	"""
+	createdAt! number!
+
+	"""
+	File Name
+	Name of attached file
+	"""
+	fileName string
+
+	"""
+	Media Type
+	Format of the attached file	
+	"""
+	mediaType string
+
+	"""
+	Url
+	Source url of the attachment
+	"""
+	url string
+
+	"""
+	Preview
+	Preview of the attached file
```

Comment:
What is this? An URL with the preview? Is it preview image? If this is the case, I suggest renaming this to `previewUrl`

Reply:
This only applies to slack and it contains string preview of attachment file.

Reply:
Okay, so looking at docs: https://api.slack.com/types/file#previews
Seems this has limited usage for text snippets. I wouldn't mind removing it. Alternatively please expand the description like "Short text preview of text attachments (e.g. snippets, posts)"

Reply:
I agree, I'll remove it.